lib/mjsonwp.js

```diff
@@ -31,7 +31,7 @@ function checkParams (params, jsonObj) {
   let notPresentParams = _.difference(sentParams, params);
   let tooPresentParams = _.difference(params, sentParams);
   if (notPresentParams.length || tooPresentParams.length) {
-    throw new BadParametersError(params, _.keys(jsonObj));
+    throw new errors.BadParametersError(params, _.keys(jsonObj));
```

Comment:
Since we want to return a `400` instead of a `500` when we have bad parameters (as per the [spec](https://code.google.com/p/selenium/wiki/JsonWireProtocol#Error_Handling)) should we check, in the `catch` block [below](https://github.com/appium/node-mobile-json-wire-protocol/blob/master/lib/mjsonwp.js#L64) that `err instance of errors.BadParametersError` and set the status accordingly?